orchestrator-salt/src/main/resources/salt/salt/unbound/init.sls

```diff
@@ -43,6 +43,13 @@ include_access_config:
     - repl: 'include: "/etc/unbound/access.conf"'
     - unless: grep "/etc/unbound/access.conf" /etc/unbound/unbound.conf
 
+set_max_ttl:
+  file.replace:
+    - name: /etc/unbound/unbound.conf
+    - pattern: '# cache-max-ttl:.*'
```

Comment:
if we want to ensure it's always has this value the pattern could look like:
`(# )?cache-max-ttl:.*`

Reply:
for extra safety also
`- append_if_not_found: True`
could be added :)